fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/AstBuilder.java

```diff
@@ -532,6 +542,12 @@ protected ParseNode visitSortItem(SortItem node, ParseTreeContext context) {
                 getNullOrderingType(isAsc, node.getNullOrdering()));
     }
 
+    @Override
+    protected ParseNode visitOffset(Offset node, ParseTreeContext context) {
+        long offset = ((LiteralExpr) visit(node.getRowCount(), context)).getLongValue();
+        return new LimitElement(offset, 0);
```

Comment:
I think use limit default value -1 is better

Reply:
and plz add some test to cover these

Reply:
done